src/app/shared/utils.ts

```diff
@@ -0,0 +1,9 @@
+import { AppBarViews, AppView } from '../layout/layout.model';
+
+export const VIEWS: AppView[] = [
```

Comment:
i think we need to move this directly into navbar component as this file is consuming a file that is inside a isolated component. Shared folder should not be requiring files from different isolated modules.